From ecf36e4e27fe1321f4aae1f0a67de3c553850bc1 Mon Sep 17 00:00:00 2001
From: Zhang Zekun <zhangzekun11@huawei.com>
Date: Mon, 24 Oct 2022 07:51:35 +0000
Subject: [PATCH 041/595] [Huawei] Renaming __find_sp_area_locked to
 find_sp_area_locked

Offering: HULK
hulk inclusion
category: bugfix
bugzilla: 187808

-----------------------------------------

Fix the following CodeCheck2.0 static check warning:
Do not declare or define reserved identifiers. The reserved identifier:
__find_sp_area_locked

Fix this by renaming __find_sp_area_locked to find_sp_area_locked.

Signed-off-by: Zhang Zekun <zhangzekun11@huawei.com>
---
 mm/share_pool.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/mm/share_pool.c b/mm/share_pool.c
index 6558cc3a99a1..5c436b3d5368 100644
--- a/mm/share_pool.c
+++ b/mm/share_pool.c
@@ -1919,7 +1919,7 @@ static struct sp_area *sp_alloc_area(unsigned long size, unsigned long flags,
 }
 
 /* the caller should hold sp_area_lock */
-static struct sp_area *__find_sp_area_locked(struct sp_group *spg,
+static struct sp_area *find_sp_area_locked(struct sp_group *spg,
 		unsigned long addr)
 {
 	struct sp_mapping *spm = sp_mapping_find(spg, addr);
@@ -1945,7 +1945,7 @@ static struct sp_area *get_sp_area(struct sp_group *spg, unsigned long addr)
 	struct sp_area *n;
 
 	spin_lock(&sp_area_lock);
-	n = __find_sp_area_locked(spg, addr);
+	n = find_sp_area_locked(spg, addr);
 	if (n)
 		atomic_inc(&n->use_count);
 	spin_unlock(&sp_area_lock);
@@ -2040,7 +2040,7 @@ void sp_area_drop(struct vm_area_struct *vma)
 	 * Considering a situation where task A and B are in the same spg.
 	 * A is exiting and calling remove_vma() -> ... -> sp_area_drop().
 	 * Concurrently, B is calling sp_free() to free the same spa.
-	 * __find_sp_area_locked() and __sp_area_drop_locked() should be
+	 * find_sp_area_locked() and __sp_area_drop_locked() should be
 	 * an atomic operation.
 	 */
 	spin_lock(&sp_area_lock);
-- 
2.17.1

